node/pkg/bus/bus.go

```diff
@@ -0,0 +1,31 @@
+package bus
+
+// message bus will be passed as parameter to modules that need to communicate with each other
+
+type Message struct {
+	From    string
+	To      string
+	Content string
+}
+
+type MessageBus struct {
+	channels map[string]chan Message
+}
+
+func NewMessageBus() *MessageBus {
+	return &MessageBus{
+		channels: make(map[string]chan Message),
+	}
+}
+
+func (mb *MessageBus) Subscribe(id string) <-chan Message {
+	ch := make(chan Message, 100)
```

Comment:
Can you please share how you decided for the channel buffer size 100?

Reply:
there is no specific reason, it could be modified anytime. maybe it'd be better to take as parameter on initialization

Reply:
Yes, I saw it somewhere else defined as well as constant.

Reply:
it's probably raft package I'll make an update there too